apps/Cargo.toml

```
@@ -81,6 +81,7 @@ tonic = "0.4.0"
 tracing = "0.1.26"
 tracing-log = "0.1.2"
 tracing-subscriber = "0.2.18"
+async-std = {version = "1.9.0", features = ["attributes", "unstable"]}
```

```suggestion
async-std = {version = "1.9.0", features = ["attributes", "unstable"]}
```
```suggestion
async-std = {version = "1.9.0", features = ["unstable"]}
```

Looks like we're not using it and it says it affects compile times